### PR TITLE
Neosec fix time format bug

### PR DESCRIPTION
**Pull request by gala-neosec**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
@gala-neosec  Please fille the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form

**Comment by arsenyk-neosec** [comment]
> 

@adi88d i submitted the form

File: Packs/Neosec/Integrations/Neosec/Neosec.py [hunk 1]
```diff
@@ -273,10 +267,14 @@ def fetch_incidents(
             if incident_created_time <= last_fetch:
                 continue
 
+        alert_timestamp = dateparser.parse(alert["timestamp"])
```
Comment (adi88d) [inline Packs/Neosec/Integrations/Neosec/Neosec.py:270]:
```suggestion
        alert_timestamp = dateparser.parse(alert.get("timestamp"))
```

**Review by adi88d — CHANGES_REQUESTED** [review]
Hi @gala-neosec and thank you for your contribution!
Good work :)

Please bump the version of the pack in the pack_metadata.json file and add notes describing your changes to a new Release Notes file. 
To do that, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.

**Review by adi88d — APPROVED** [review]
nice!

**Pull request merged**